'unresolved external symbol' error. VS2015.

I can't build one project under the Visual Studio 2015 (Version 14.0.23107.0 D14REL) due to 'nanodbc' 2.12.4 version. 

I get  'unresolved external symbol' like this - 

`Error   LNK2001   unresolved external symbol "__declspec(dllimport) public: static class std::locale::id std::codecvt<char16_t,char,struct _Mbstatet>::id" (__imp_?id@?$codecvt@_SDU_Mbstatet@@@std@@2V0locale@2@A)   33_Urho2DSpriterAnimation   Github\Urho3D-master\build_1\Source\Samples\33_Urho2DSpriterAnimation\Urho3D_d.lib(nanodbc.obj)   1`

I have found out that is bug in vs2015. 

I see that in 'nanodbc' code is some workaround, but it seems do not work in my case. 

```
// Workaround for confirmed bug in VS2015.
// See: https://social.msdn.microsoft.com/Forums/en-US/8f40dcd8-c67f-4eba-9134 a19b9178e481/vs-2015-rc-linker-stdcodecvt-error

auto p = reinterpret_cast<wide_char_t const*>(in.data());
out = std::wstring_convert<NANODBC_CODECVT_TYPE<wide_char_t>, wide_char_t>().to_bytes(p, p + in.size());
```

UPDATE: 

I have tried to build nanodbc (static and dynamic)  under vs2015 as a standalone using its own build system. The result is the same.

If I build nanodbc as static, I get the 'unresolved external symbol' error - 
`Error   LNK2001   unresolved external symbol "__declspec(dllimport) public: static class std::locale::id std::codecvt<char16_t,char,struct _Mbstatet>::id" (__imp_?id@?$codecvt@_SDU_Mbstatet@@@std@@2V0locale@2@A)   example_northwind   D:\nanodbc-master\build\examples\nanodbc.lib(nanodbc.obj)   1`

If I build nanodbc as dynamic, I get the 'unresolved external symbol' error - 
`nanodbc.obj : error LNK2001: unresolved external symbol "__declspec(dllimport) public: static class std::locale::id std::codecvt<char16_t,char,struct _Mbstatet>::id" (__imp_?id@?$codecvt@_SDU_Mbstatet@@@std@@2V0locale@2@A)`

 In between times, I changed cmakelists.txt line 206 - 
`install(TARGETS nanodbc LIBRARY DESTINATION lib)` 
to
`install (TARGETS nanodbc
         ARCHIVE DESTINATION lib
         LIBRARY DESTINATION lib
         RUNTIME DESTINATION bin)`

in order to generate project as dynamic. (I use cmake 3.5.0)

--- a/src/nanodbc.cpp
+++ b/src/nanodbc.cpp
@@ -209,10 +209,11 @@ namespace
             out = utf_to_utf<char>(in.c_str(), in.c_str() + in.size());
         #else
             #if defined(_MSC_VER) && (_MSC_VER == 1900)
-                // Workaround for confirmed bug in VS2015.
-                // See: https://social.msdn.microsoft.com/Forums/en-US/8f40dcd8-c67f-4eba-9134-a19b9178e481/vs-2015-rc-linker-stdcodecvt-error
-                auto p = reinterpret_cast<wide_char_t const*>(in.data());
-                out = std::wstring_convert<NANODBC_CODECVT_TYPE<wide_char_t>, wide_char_t>().to_bytes(p, p + in.size());
+                // Workaround for confirmed bug in VS2015. See:
+                // https://connect.microsoft.com/VisualStudio/Feedback/Details/1403302
+                // https://social.msdn.microsoft.com/Forums/en-US/8f40dcd8-c67f-4eba-9134-a19b9178e481/vs-2015-rc-linker-stdcodecvt-error
+                auto p = reinterpret_cast<unsigned short const*>(in.data());
+                out = std::wstring_convert<NANODBC_CODECVT_TYPE<unsigned short>, unsigned short>().to_bytes(p, p + in.size());
             #else
                 out = std::wstring_convert<NANODBC_CODECVT_TYPE<wide_char_t>, wide_char_t>().to_bytes(in);
             #endif
@@ -226,9 +227,10 @@ namespace
                 using boost::locale::conv::utf_to_utf;
                 out = utf_to_utf<wide_char_t>(in.c_str(), in.c_str() + in.size());
             #elif defined(_MSC_VER) && (_MSC_VER == 1900)
-                // Workaround for confirmed bug in VS2015.
-                // See: https://social.msdn.microsoft.com/Forums/en-US/8f40dcd8-c67f-4eba-9134-a19b9178e481/vs-2015-rc-linker-stdcodecvt-error
-                auto s = std::wstring_convert<NANODBC_CODECVT_TYPE<wide_char_t>, wide_char_t>().from_bytes(in);
+                // Workaround for confirmed bug in VS2015. See:
+                // https://connect.microsoft.com/VisualStudio/Feedback/Details/1403302
+                // https://social.msdn.microsoft.com/Forums/en-US/8f40dcd8-c67f-4eba-9134-a19b9178e481/vs-2015-rc-linker-stdcodecvt-error
+                auto s = std::wstring_convert<NANODBC_CODECVT_TYPE<unsigned short>, unsigned short>().from_bytes(in);
                 auto p = reinterpret_cast<wide_char_t const*>(s.data());
                 out.assign(p, p + s.size());
             #else

